blocks/bricks/conv.py

```diff
@@ -177,6 +180,59 @@ def num_output_channels(self):
         return self.num_filters
 
 
+class ConvolutionalTranspose(Convolutional):
+    """Performs the transpose of a 2D convolution.
+
+    Parameters
+    ----------
+    image_size : tuple, optional
```

Comment:
I think this should go last if anywhere. Probably also should note that this is the input to the *transpose*, output of the original convolution.